src/components/IssueList.tsx

```diff
@@ -279,11 +320,40 @@ export default function IssueList(props: {title: title; box: box}) {
                                 name={`issues.${index}.notes`}
                                 className="border"
                                 type="text"
-                                disabled={newspaperIsSaved(index, values.issues.length)}
+                                disabled={shouldDisableIssue(index, values.issues.length)}
                                 value={issue.notes || ''}
                               />
                             </TableCell>
                             <TableCell className="text-lg">
+                              {newspaperIsSaved(index, values.issues.length) &&
+                                <>
+                                  {isEditingIssue(index) ?
+                                    <>
+                                      {issueBeingSaved ?
+                                        <Spinner size='sm' className='mr-2'/>
+                                        :
+                                        <>
+                                          <button className='mr-2' type='button' onClick={() => updateIssue(issue)}>
+                                            <FaSave color='1ba636'/>
+                                          </button>
+                                          <button className='mr-2' type='button' onClick={() => stopEditingIssue()}>
+                                            <ImCross color='dca238'/>
+                                          </button>
+                                        </>
+                                      }
+                                    </>
+                                    :
+                                    <button
+                                      className={isEditingIssue() ? 'mr-2 opacity-30' : 'mr-2'}
+                                      type='button'
+                                      disabled={isEditingIssue()}
+                                      onClick={() => startEditingIssue(index)}
+                                    >
+                                      <FiEdit color='blue'/>
+                                    </button>
```

Comment:
Ikke så viktig, men kunne brukt Next UI for icon-only-knapper for mer likhet med resten av knappene: https://nextui.org/docs/components/button#icon-only